### ISSUE TITLE
Issue with slim/scripts/finetune_resnet_v1_50_on_flowers_CUSTOM.sh

### ISSUE THREAD

**Issue by dimtziwnas** [tensorflow/models#1784]
I try to run the provided script (TF 1.2.0, gpu enabled, pip install) for finetuning resnet (similar issues for inception scripts).

The model- and dataset-downloading is performed without a problem. However at the next step, when fine-tuning is performed, i.e.:

    # Fine-tune only the new layers for 3000 steps.
    python train_image_classifier.py \
      --train_dir=${TRAIN_DIR} \
      --dataset_name=flowers \
      --dataset_split_name=train \
      --dataset_dir=${DATASET_DIR} \
      --model_name=resnet_v1_50 \
      --checkpoint_path=${PRETRAINED_CHECKPOINT_DIR}/resnet_v1_50.ckpt \
      --checkpoint_exclude_scopes=resnet_v1_50/logits \
      --trainable_scopes=resnet_v1_50/logits \
      --max_number_of_steps=3000 \
      --batch_size=32 \
      --learning_rate=0.01 \
      --save_interval_secs=60 \
      --save_summaries_secs=60 \
      --log_every_n_steps=100 \
      --optimizer=rmsprop \
      --weight_decay=0.00004

there is the following issue:

     INFO:tensorflow:Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.InvalidArgumentError'>, Cannot assign a device for operation 'resnet_v1_50/logits/biases/RMSProp_1': Operation was explicitly assigned to /device:GPU:0 but available devices are [ /job:localhost/replica:0/task:0/cpu:0 ]. Make sure the device specification refers to a valid device.
	 [[Node: resnet_v1_50/logits/biases/RMSProp_1 = VariableV2[_class=["loc:@resnet_v1_50/logits/biases"], container="", dtype=DT_FLOAT, shape=[5], shared_name="", _device="/device:GPU:0"]()]]

My GPU is detected without a problem though when I run:

     sess = tf.Session(config=tf.ConfigProto(log_device_placement=True))

**Comment by dimtziwnas** [comment]
Issue was solved with reinstalling TF, probably something was affected accidentally.
